src/app/search/directives/search-story.component.css

```diff
@@ -0,0 +1,121 @@
+.story-image {
+  position: relative;
+  float: left;
+  width: 120px;
+  height: 120px;
+  margin-bottom: 5px;
+}
+image-loader {
+  width: 100%;
+  height: 100%;
+  display: block;
+}
+
+.status {
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding: 3px 10px;
+  z-index: 10;
+  color: #fff;
+  background-color: #444;
+  font-weight: 400;
+  font-size: 11px;
+  text-transform: uppercase;
+  line-height: 18px;
+}
+.status.unpublished {
+  background-color: #ee4d3e;
+}
+
+.story-detail {
+  position: relative;
+  float: right;
+  padding: 10px;
+  width: 200px;
+  height: 120px;
+  line-height: 20px;
+}
+.story-title {
+  font-size: 16px;
+  max-height: 40px;
+  overflow: hidden;
+  text-overflow: ellipsis;
```

Comment:
This is a lie. Multiline truncation won't ellipsis with text-overflow: ellipsis alone. Do we even want ellipsis in the titles? Two lines seems like enough. I was primarily concerned with overflowing.